src/Order.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = ({ orderType }) => {
+  const [orders, setOrders] = useState(0);
+
+  const ClickHandler = () => {
```

Comment:
Any reason why this has a capital letter to start? Remember we should use camelCase for all our variables and functions in JS. With the exception of a React component, who we do use a capital letrer.